compiler/quilt/tools/command.py

```diff
@@ -648,19 +648,21 @@ def log(package):
             str(entry.get('tags', [])), str(entry.get('versions', []))))
     _print_table(table)
 
-def push(package, is_public=False, is_team=False, reupload=False):
+def push(package, hash=None, is_public=False, is_team=False, reupload=False):
```

Comment:
Make it the last parameter, just in case someone uses the API with non-keyword args.

Reply:
Sure. I was trying to match install, but I think you're right. Better to make it the last arg to avoid breaking existing code.